emag-tech-labs/rabbitmq-bundle/2.0/config/packages/old_sound_rabbit_mq.yaml

```diff
@@ -0,0 +1,17 @@
+old_sound_rabbit_mq:
+    connections:
+        default:
+            url: '%env(RABBITMQ_URL)%'
+            vhost: '%env(RABBITMQ_VHOST)%'
+            lazy:     false
+            connection_timeout: 3
+            read_write_timeout: 3
+
+            # requires php-amqplib v2.4.1+ and PHP5.4+
+            keepalive: false
+
+            # requires php-amqplib v2.4.1+
+            heartbeat: 0
+
+            #requires php_sockets.dll
+            use_socket: true # default false
```

Comment:
`php_sockets.dll`? Isn't that windows only?

Why is this true by default?

Reply:
@andreea-anamaria let's put it to false

Reply:
Fixed.